arbitrator/tools/stylus_benchmark/src/generate_wats.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021-2024, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro/blob/master/LICENSE
+
+use std::fs::File;
+use std::io::Write;
+use std::path::PathBuf;
+
+fn generate_add_i32_wat(mut out_path: PathBuf) -> eyre::Result<()> {
```

Comment:
the process of generating wat and then executing it is cumbersome..
we could enable it for debug purposes, but normal execution should just state which benchmark you want, it will be generated as wat, translated with wat2wasm and executed without being written to filesystem.

Reply:
OK :)
I will change the UX to be something similar to:
`stylus_benchmark --output_wat --scenario i32_add`
`--output_wat` and `--scenario SCENARIO` will be optional.
If `--scenario SCENARIO` is not provided then all scenarios are going to be benchmarked.

Reply:
Done.